packages/core/admin/_internal/node/develop.ts

```diff
@@ -47,6 +84,12 @@ const develop = async ({
       return;
     }
 
+    if (tsconfig) {
```

Comment:
According to other places where it's checked, it seems like the check is done `tsconfig.config`. But I have no idea if it changes smth or not

Reply:
Done